packages/mui-material/src/Divider/dividerClasses.ts

```diff
@@ -10,7 +10,9 @@ export interface DividerClasses {
   inset: string;
   /** Styles applied to the root element if `variant="fullWidth"`. */
   fullWidth: string;
-  /** Styles applied to the root element if `light={true}`. */
+  /** Styles applied to the root element if `light={true}`.
+   * @deprecated
```

Comment:
i'm not sure, what's the alternative way for `light` class selector. so just left empty as of now